src/Benchmarks/Data/DapperDb.cs

```diff
@@ -58,6 +58,28 @@ public DapperDb(IRandom random, DbProviderFactory dbProviderFactory, IOptions<Ap
             return result;
         }
 
+        public async Task<World[]> LoadMultipleUpdatesRows(int count)
+        {
+            var result = await LoadMultipleQueriesRows(count);
+
+            using (var db = _dbProviderFactory.CreateConnection())
+            {
+                db.ConnectionString = _connectionString;
+                await db.OpenAsync();
+
+                for (int i = 0; i < count; i++)
```

Comment:
I guess it depends on if dapper is considered an orm.  The scenario doc says for raw accesses to do individual queries and updates and an orm can do whatever it does under the covers.  The current dapper implementation does individual queries for the multiple queries scenario and dapper doesn't have first class support for updates, so multiple update queries seemed to be in the spirit of the scenario.